### ISSUE TITLE
Dropdown: arrow navigation

### ISSUE THREAD

**Issue by Magnusrm** [digdir/designsystemet#3744]
### Description

When using the dropdown in Altinn forms we have gotten a request from UDIR to have arrow navigation in the dropdown instead of / in addition to tab-navigation.

UDIR interprets this as a WCAG violation.

![Image](https://github.com/user-attachments/assets/9ece9d15-407e-4612-bccb-fc486b6072a3)

Are there some reasons to not have arrow navigation?

**Comment by Barsnes** [comment]
Hi,
our `Dropdown` is not a menu, so we won't follow these rules in our component. You will need to implement this yourself.

If you need help to implement arrow navigation, take a look at our `RovingFocus` utility component:
https://storybook.designsystemet.no/?path=/docs/utilities-rovingfocus--docs
